### PR TITLE
Add rights field to generated updateinfo.xml

### PR DESCRIPTION
**Pull request by mackdk**
## What does this PR change?

This PR retrieves and store the copyright information from the remote repository in order to add it to the updateinfo.xml of the generated repository. 

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [X] **DONE**

## Test coverage
- Unit tests were added

- [X] **DONE**

## Links

Fixes SUSE/spacewalk/issues/12198

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: schema/spacewalk/common/tables/rhnErrata.sql [hunk 1]
```diff
@@ -36,6 +36,7 @@ CREATE TABLE rhnErrata
     synopsis          VARCHAR(4000) NOT NULL,
     topic             VARCHAR(4000),
     solution          VARCHAR(4000) NOT NULL,
+    rights            VARCHAR(100),
```
Comment (mcalmer) [inline schema/spacewalk/common/tables/rhnErrata.sql:39]: I wonder if "100" is enough. Did you check what is used out there?

**Review by meaksh — APPROVED** [review]
Python part looks good :+1:

File: schema/spacewalk/common/tables/rhnErrata.sql (hunk 1 above)
Comment (mackdk) [reply schema/spacewalk/common/tables/rhnErrata.sql:39]: Yes, all that I saw is a variation of "Copyright <year> <company>". The longest I tested is from RockyLinux "Copyright (C) 2021 Rocky Enterprise Software Foundation" which is 55 characters. But you have a point, maybe I can bump it to 200 to be extra safe.

Comment (mcalmer) [reply schema/spacewalk/common/tables/rhnErrata.sql:39]: Ok, then 100 should be good enough

**Review by mcalmer — APPROVED** [review]

**Review by lucidd — APPROVED** [review]

**Comment by juliogonzalez** [comment]
@mackdk did you run the acceptance tests on PR?

**Comment by juliogonzalez** [comment]
Talked to @mackdk on slack. CI seems to be broken for PRs and the PRS "are all new features I don't see any urgency". Removing merge-candidate label for now. Feel free to add it again later, if the tests are OK and the branch is still blocked.

**Pull request merged**